packages/reactive-element/package.json

```diff
@@ -45,20 +45,20 @@
   "scripts": {
     "build": "npm run clean && npm run build:ts && rollup -c && npm run build:babel",
     "build:watch": "rollup -c --watch",
-    "build:ts": "tsc --build && treemirror development . '**/*.d.ts{,.map}'",
```

Comment:
Yea windows and unix treat single quotes differently, think it ends up as `"'quoted text'"` rather than the expected `"quoted text"`, also think double quotes are needed for globs on windows.

In this case the single quoted version gives this error: `'html)'' is not recognised as an internal or external command, operable program or batch file.`

Reply:
TIL thanks!